product-backend/src/main/java/org/kainos/ea/exception/FailedToCreateBandException.java

```diff
@@ -0,0 +1,13 @@
+package org.kainos.ea.exception;
+
+public class FailedToCreateBandException extends Throwable {
+
+    public FailedToCreateBandException(String s) {
+        super(s);
+    }
+
+    @Override
+    public String getMessage() {
+        return "Failed to create band";
+    }
```

Comment:
This combination of constructor and getMessage override makes no sense. You should probably remov the constructor. Here and in the other exception classes.

Reply:
Fixed.